### ISSUE TITLE
[BUG] Bytes property comparisons not implemented

### ISSUE THREAD

**Issue by philippjfr** [bokeh/bokeh#13076]
### Software versions

Bokeh: 3.1.0

### Expected behavior

When creating a Model with a `Bytes` property events should be triggered appropriately if/when the value changes.

### Observed behavior

The `Comparator` errors out because no comparison is defined for an object of type `ArrayBuffer`:

https://github.com/bokeh/bokeh/blob/5190f2455fa7115283e121c72d71e061cbffea08/bokehjs/src/lib/core/util/eq.ts#L81

### Example code

```Python
None
```

### Stack traceback or browser console output

> Uncaught Error: can't compare objects of type [object ArrayBuffer]

**Comment by mattpap** [comment]
> `Uncaught Error: can't compare objects of type [object ArrayBuffer]`

This is correct behavior. Array buffers are opaque memory containers, and as such they can be only implicitly compared (e.g. via `uint8` views). Stray array buffers should not appear as property values. Instead they should be wrapped in a typed array or ndarray, etc. I'm pretty sure this is either a bug or bad design somewhere else in bokeh or bokehjs. An MRE would be useful to investigate the root cause.

**Comment by philippjfr** [comment]
This wasn't in bokeh, I created some models in Panel that shuffle across raw bytes (usually zipped arrays for VTK).

**Comment by bryevdv** [comment]
@philippjfr @mattpap what is the status here? Without an MRE I think this should be closed.
